src/EndpointConfig.js

```diff
@@ -7,11 +7,13 @@ const urlMap = {
   'testnet-edge': 'http://edge.testnet.solana.com:8899',
   'testnet-beta': 'http://beta.testnet.solana.com:8899',
   testnet: 'http://testnet.solana.com:8899',
+  'testnet-tds': 'http://tds.solana.com:8899',
```

Comment:
wanna just call this `tds` (drop the `testnet-` prefix)

Reply:
Should I style it `TdS` in that case?

Reply:
lowercase for now, we also use this value to index into metrics.  We prolly need to add a friendly name field soon...